README.rst

```diff
@@ -1049,6 +1049,3 @@ limitations under the License.
    :target: https://github.com/falconry/falcon/actions?query=workflow%3A%22Run+tests%22
 .. |codecov.io| image:: https://codecov.io/gh/falconry/falcon/branch/master/graphs/badge.svg
    :target: http://codecov.io/gh/falconry/falcon
-.. |Blue| image:: https://img.shields.io/badge/code%20style-blue-blue.svg
```

Comment:
ruff has no badge?

Reply:
None of the referenced major projects has any ruff badge, so I thought we can wait with this :)